app/views/admin/legal_aid_applications/submissions/show.html.erb

```diff
@@ -65,7 +65,11 @@
           CFE result
         </summary>
         <section>
-          <%= format_hash(JSON.parse(cfe_submission.result&.result || empty_cfe_result)) %>
```

Comment:
can the `empty_cfe_result` result be deleted from line 2 of this file. it does not appear to be used anywhere now